agent/api/appmesh/appmesh.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+package appmesh
+
+import (
+	"github.com/aws/amazon-ecs-agent/agent/acs/model/ecsacs"
```

Comment:
The convention that we normally follows for import is 
import (
  "standard package"
  +empty line
  "third-party package"
)
e.g. import (
   "fmt"
  
   "github.com/****"
)

Reply:
will do :)